skyportal/handlers/api/candidate.py

```diff
@@ -1199,15 +1205,6 @@ def grab_query_results(
             .join(obj_ids_values, obj_ids_values.c.id == Obj.id)
             .order_by(obj_ids_values.c.ordering)
         )
-    else:
-        # If there are no values, the VALUES statement above will cause a syntax error,
-        # so just switch to this simple IN query. Should return nothing either way
-        items = (
-            DBSession()
-            .query(Obj)
-            .options(query_options)
-            .filter(Obj.id.in_(obj_ids_in_page))
-        )
 
     info[items_name] = items
```

Comment:
Wouldn't this error in the `else` case (items undefined)?

Reply:
`items` is actually initialized to `[]` already earlier (line 1194 in the newest commit)

Reply:
Oops, missed that. In it goes then!